contracts/SimplexVerifier.sol

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/// @title SimplexVerifier
+/// @notice Verifies deserialization of Simplex consensus proofs
+contract SimplexVerifier {
+    // Constants for proof sizes
+    uint256 constant DIGEST_LENGTH = 32; // SHA256 digest length
```

Comment:
I think we might want to provide these via some initializer? It is possible that Simplex is instantiated with a variety of different schemes.